Makefile

```diff
@@ -8,6 +8,8 @@ IMAGE_NAME ?= ceph-csi-operator
 NAME_PREFIX ?= ceph-csi-operator-
 NAMESPACE ?= $(NAME_PREFIX)system
 
+
+
```

Comment:
perfect can you please remove this extra two new lines?

Reply:
gone